Makefile

```diff
@@ -119,6 +119,8 @@ endif
 
 #kubernetes demo targets
 k8s-cluster:
+	cd vagrant/k8s/ && CONTIV_k8=1 ./setup_cluster.sh
```

Comment:
caps required for  CONTIV_k8=1 ?
if it is using kubeadm which is different from the legacy install, can you make that part of the flag ?